packages/calypso-e2e/src/lib/components/sidebar-component.ts

```diff
@@ -86,15 +104,24 @@ export class SidebarComponent extends BaseContainer {
 			// This works better than using CSS pseudo-classes like `:has-text` or `:text-matches` for text
 			// matching.
 			selector = `${ selectors.subheading } >> text="${ subitem }"`;
+			await this._click( selector );
 		}
+	}
 
-		if ( ! selector ) {
-			throw new Error(
-				`Selector is undefined. Check if item or subitem has been specified as argument(s).`
-			);
-		}
-
-		await this._click( selector );
+	/**
+	 * Opens the sidebar into view.
```

Comment:
Would be good to note that it's true for mobile viewport only

Reply:
Done.